RideShare/src/app/services/HousingLocationService/housing-location.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { TrainingLocation } from "../../models/traininglocation.model";
+import { HouseLocation } from "../../models/houselocation.model";
+
+@Injectable({
+  providedIn: "root"
+})
+
+/**
+ * HousingLocationService gets all housing locations
+ *
+ * Imported by:
+ */
+export class HousingLocationService {
+  /**
+   * Server and port number of the request
+   */
+  readonly port = "http://localhost:3003";
```

Comment:
move to environment

Reply:
Changed to environment variable in environments.ts.